src/status_im2/contexts/wallet/send/select_asset/style.cljs

```diff
@@ -0,0 +1,19 @@
+(ns status-im2.contexts.wallet.send.select-asset.style)
+
+(defn container
+  [margin-top]
+  {:flex       1
+   :margin-top margin-top})
+
+(def title-container
+  {:margin-horizontal 20
+   :margin-vertical   12})
+
+(def empty-container-style
+  {:justify-content :center
+   :flex            1
+   :margin-bottom   44})
```

Comment:
hmm, can this be achieved with safe-area? 🤔  or the page insets set on the screen.cljs 🤔

Reply:
Actually I copied the style from other place but replaced to use safe area inset